app/install/osx.md

```diff
@@ -9,18 +9,20 @@ breadcrumbs:
 
 ### Packages
 
-- [.pkg Installer]({{ site.links.download }}/{{site.data.kong_latest.version}}/kong-{{site.data.kong_latest.version}}.osx.pkg)
 - [Homebrew Formula]({{ site.repos.homebrew }})
 
 ----
 
 ### Installation
 
-1. **Install Kong**
+Use the [Homebrew](https://brew.sh/) package manager to add Kong as a tap and install it:
```

Comment:
Wouldn't say out of context; before the actual installation steps to make sure the user is familiar with it before jumping into it?